### ISSUE TITLE
Rendered JPEG appears rotated.

### ISSUE THREAD

**Issue by PepijnK** [disintegration/imaging#43]
I have some pictures shot in portrait position coming from my digital camera (Panasonic Lumix) and phone (Samsung S7) which appear rotated after loading, fitting and saving them.
Here is the code:

```
package main

import (
    "flag"
    "fmt"

    "github.com/disintegration/imaging"
)

func main() {
    filename := flag.String("file", "", "input file")
    flag.Parse()
    if filename == nil {
        panic("missing input file")
    }

    fmt.Println("opening " + *filename)

    img, err := imaging.Open(*filename)
    if err != nil {
        panic(err)
    }

    dst := imaging.Fit(img, 800, 600, imaging.Lanczos)

    if err := imaging.Save(dst, "converted.jpg"); err != nil {
        panic(err)
    }
}
```

The original is (when downloaded) shown in portrait mode.
![original](https://cloud.githubusercontent.com/assets/3328273/19727743/f7b1ab1e-9b90-11e6-9268-528fa752971f.jpg)

The converted (probably has all EXIF info scrapped) is shown in neutral mode (guess that's landscape)
![converted](https://cloud.githubusercontent.com/assets/3328273/19727756/006ea108-9b91-11e6-939e-59afae30202c.jpg)

**Comment by disintegration** [comment]
Hello,

Your example images both appear landscape-mode to me, but you're right about the exif info. Go's `image` package does not support exif info loading/saving. The `image/jpeg` package doesn't take the exif info into consideration when loading images.

You can try using some other package to read/write the exif data separately or just to read the orientation exif tag from jpeg and fix the image orientation after loading using imaging flip and rotate functions.

See also: https://github.com/disintegration/imaging/issues/30

**Comment by PepijnK** [comment]
OK, I will check that. Are you thinking about providing that functionality as well? If you click the images, you see what I mean (apparently GitHub doesn't support resized EXIF'd pictures as well! Maybe they're using your lib? ;) )

**Comment by disintegration** [comment]
I don't want to change the behavior of the `imaging.Open` function as a simple wrapper around `image.Decode`.

But this functionality can be implemented in a separate function. I didn't think about it yet. Probably I will look into it later. Also I would like not to introduce another external dependency.

**Comment by PepijnK** [comment]
Fixed this with instruction from #30
